ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -49,12 +48,11 @@ public String type() {
 
     @Value.Check
     protected final void check() {
-        checkArgument(!path().equals(Paths.get("")), "path cannot be empty");
```

Comment:
sorry, missed this first time through, I think we want to error if the URI contains the empty string, can we have a check for that somehow?